uwblueprint/focus-on-nature
# feat: Camp overview page - edit camper information

* Added menu to camp overview registration table

## Notion ticket link

[Camp Overview: Edit Camper Information](https://www.notion.so/uwblueprintexecs/Camp-Overview-Edit-Camper-Information-e1dd8cd472ef4ad6a2de3cf1026e01fa)



## Implementation description
* Added a new component called `CampersTableMenu` for the dropdown menu in the registered campers table 



## Steps to test
1. go to http://localhost:3000/admin/camp/63139c7bc3d7b55b44a01531 (this is the url to access the Guelph Camp overview page)
2. click on the 3 dot menu to see all the menu options



## What should reviewers focus on?
* Design specifications 
* How do you make the menu only render downwards towards the bottom? It always renders upwards for me :( 


## Checklist
